test/JDBC/input/storedProcedures/Test-sp_helpsrvrolemember-vu-verify.sql

```diff
@@ -0,0 +1,17 @@
+EXEC test_sp_helpsrvrolemember_proc
```

Comment:
we also want to have tests which validate that this function/proc can be part of another proc/function/view to make sure dependencies are taken care of. As it is a new function, corresponding prepare should only run on latest but become an upgrade test in future